src/psbt.h

```diff
@@ -56,68 +184,109 @@ struct PSBTInput
     CScriptWitness final_script_witness;
     std::map<CPubKey, KeyOriginInfo> hd_keypaths;
     std::map<CKeyID, SigPair> partial_sigs;
+    uint256 prev_txid;
+    std::optional<uint32_t> prev_out;
+    std::optional<uint32_t> sequence;
+    std::optional<uint32_t> time_locktime;
+    std::optional<uint32_t> height_locktime;
     std::map<std::vector<unsigned char>, std::vector<unsigned char>> unknown;
+    std::set<PSBTProprietary> m_proprietary;
     int sighash_type = 0;
 
+    uint32_t m_psbt_version;
+
     bool IsNull() const;
     void FillSignatureData(SignatureData& sigdata) const;
     void FromSignatureData(const SignatureData& sigdata);
     void Merge(const PSBTInput& input);
-    PSBTInput() {}
+    bool GetUTXO(CTxOut& utxo) const;
```

Comment:
Should the [original comment](https://github.com/bitcoin/bitcoin/pull/21283/commits/7d6ac06858d5f15de5fd86b2c055efd6c3bc8345#diff-2fb519d85c6ae99cd87c94e5b7cba43ff2ddd031b2c523dd2ffb12e991c16494L748) be preserved in some way?